### PR TITLE
STY: Update flake8 requirements

### LINKED ISSUE #846: flake8 requirements have changed
Some of the builds are failing because flake8 has been updated and is now flagging things it wasn't before. To reproduce this problem, upgrade your version of flake8 and run it in your repo or have a look at these builds:

- [build 1](https://travis-ci.org/biocore/qiita/builds/50422108).
- [build 2](https://travis-ci.org/biocore/qiita/builds/50420752).

### PR DESCRIPTION
**Pull request by ElDeveloper**
Fixes #846

### CODE REVIEW

**Comment by squirrelo** [comment]
Good sir you have beaten me. I concede. :+1:

**Comment by ElDeveloper** [comment]
https://www.youtube.com/watch?v=TAryFIuRxmQ

File: qiita_ware/wrapper.py [hunk 1]
```diff
@@ -17,7 +10,15 @@
 import networkx as nx
 from moi.job import system_call, submit, ctxs, ctx_default
 
-from qiita_db.job import Job
+
```
Comment (josenavas) [inline qiita_ware/wrapper.py]: I think these lines can be removed as they're leftover of a copy-paste.

**Comment by josenavas** [comment]
Thanks @ElDeveloper Just one minor comment.

Just for clarity, the imports should not be organized now as:
```
std lib imports
<new line>
third party lib imports
<new line>
project imports
```
?

Which is now the rule for the imports? In order to save some developer time...

**Comment by squirrelo** [comment]
The problem it's yelling about is that any imports after the `with .....` import are considered to be out of order. We can keep that same order we are using now and just move the `with` import to the bottom of that list.

**Comment by ElDeveloper** [comment]
If you see the links in #824, flake8 was not very clear about this, but
it consistently complained about the module-imports not being at the top
of the file.  And for some reason it no longer likes *some* lambdas ...
weird.

**Comment by ElDeveloper** [comment]
@squirrelo, you are right, I'll change that right away!

**Comment by squirrelo** [comment]
:+1:

**Pull request merged**